agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -948,3 +949,69 @@ func testV3TaskEndpointTags(t *testing.T, taskName, containerName, networkMode s
 	exitCode, _ := task.ContainerExitcode(containerName)
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+func TestContainerOrdering(t *testing.T) {
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.26.0")
+
+	td, err := GetTaskDefinition("container-ordering")
+	if err != nil {
+		t.Fatalf("Could not register task definition: %v", err)
+	}
+
+	var testTasks []*TestTask
+	testTasks, err = agent.StartMultipleTasks(t, td, 1)
+	if err != nil {
+		t.Fatalf("Could not start task: %v", err)
+	}
+
+	timeout, err := time.ParseDuration("5m")
```

Comment:
would 5m be too long for functional test since our overall timeout is 40m from [here](https://github.com/aws/amazon-ecs-agent/blob/master/Makefile#L202)

Reply:
Cool. I have made it 3m. I have 6 containers in my task def, that is why increased the timeout a bit